<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-9FISPP1M" failures="1" tests="18" name="qa.testcases.AllTest" time="141.361" errors="0" timestamp="2022-11-30T22:01:05 EST" skipped="0">
  <testcase classname="qa.testcases.AllTest" name="loginTest_sucess" time="4.888"/>
  <testcase classname="qa.testcases.AllTest" name="loginTest_failure" time="9.179"/>
  <testcase classname="qa.testcases.AllTest" name="checkboxSelectionTest" time="4.991"/>
  <testcase classname="qa.testcases.AllTest" name="contextMenu_rightClickTest" time="6.303"/>
  <testcase classname="qa.testcases.AllTest" name="drag_and_drop_Test" time="6.255"/>
  <testcase classname="qa.testcases.AllTest" name="dropDownTest" time="5.804"/>
  <testcase classname="qa.testcases.AllTest" name="dynamicContentTest" time="9.787"/>
  <testcase classname="qa.testcases.AllTest" name="dynamicControlTest" time="14.738"/>
  <testcase classname="qa.testcases.AllTest" name="dynamicLoadingTest" time="8.380"/>
  <testcase classname="qa.testcases.AllTest" name="fileDownloadTest" time="7.300"/>
  <testcase classname="qa.testcases.AllTest" name="fileUploadTest" time="4.230"/>
  <testcase classname="qa.testcases.AllTest" name="floatingMenuTest" time="5.597"/>
  <testcase classname="qa.testcases.AllTest" name="iFrameTest" time="26.308"/>
  <testcase classname="qa.testcases.AllTest" name="mouseHoverTest" time="5.142"/>
  <testcase classname="qa.testcases.AllTest" name="javaScriptAlertTest" time="7.520"/>
  <testcase classname="qa.testcases.AllTest" name="javaScriptErrorTest" time="1.866">
    <failure message="The following asserts failed:
	expected [Cannot read property &#039;xyz&#039; of undefined.] but found [This page has a JavaScript error in the onload event. This is often a problem to using normal Javascript injection techniques.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Cannot read property 'xyz' of undefined.] but found [This page has a JavaScript error in the onload event. This is often a problem to using normal Javascript injection techniques.]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at qa.testcases.AllTest.javaScriptErrorTest(AllTest.java:342)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- javaScriptErrorTest -->
  <testcase classname="qa.testcases.AllTest" name="newTabTest" time="5.602"/>
  <testcase classname="qa.testcases.AllTest" name="notificationTest" time="7.471"/>
</testsuite> <!-- qa.testcases.AllTest -->
